src/foundations/sass/_colors.scss

```diff
@@ -24,15 +24,16 @@ $btn-primary-bg: var(--btn-primary-bg, $primary-color) !default;
 =                 Navbar                      =
 ============================================= */
 
-$navbar-color: var(--navbar-color,
-    $secondary-color) !default;
+$navbar-color: var(--navbar-color, $secondary-color) !default;
 
 /* =============================================
-=                 DeliveryOrderCard                      =
+=                 DeliveryOrderCard            =
 ============================================= */
 
-$delivery-order-card-color: var(--delivery-order-card-color,
-    $secondary-color) !default;
+$delivery-order-card-color: var(
+  --delivery-order-card-color,
```

Comment:
Please revert unnecessary style changes. You can use `npm run format:fix` instead of your IDE formatter.